docs/src/pages/configurations/options-parameter/index.md

```diff
@@ -86,9 +88,65 @@ addParameters({
     storySort: (a, b) =>
       a[1].kind === b[1].kind ? 0 : a[1].id.localeCompare(b[1].id, undefined, { numeric: true }),
   },
-};
+});
+```
+
+The `storySort` can also accept a configuration object.
+
+```js
+import { addParameters, configure } from '@storybook/react';
+
+addParameters({
+  options: {
+    storySort: {
+      method: 'alphabetical', // Optional, defaults to 'configure'.
+      sort: ['Intro', 'Components'], // Optional, defaults to [].
```

Comment:
I see the PR merged in `next-6.0.0`, but not seeing the changes reflected there. I wanted to point out the `sort` key here is wrong. The `storySort` method in this PR uses `order` to override the order.

Reply:
Cc @ndelangen

Reply:
Yeah, this doc seems to be outdated. There's `order` in tests.

Reply:
@hasparus would you be able to make a PR fixing the docs for this?

Reply:
yup, no biggie